ministryofjustice/correspondence_tool_staff
# CT-3151 Complaints case editing

## Description
This PR implements the editing features for complaints cases, including the page counts but excluding the "close case" journey as this is a separate feature spec we'll come onto next.

Changes to the Show page include hiding the "clearance levels" section and displaying the correct history section, data requests UI and so on. 

## Self-review checklist


### Screenshots

![image](https://user-images.githubusercontent.com/1161161/95961337-cb614000-0dfc-11eb-979e-4810c4ba05b1.png)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3151

### Deployment
nope

### Manual testing instructions
Create an Offender SAR Complaint case and verify that the "Change" links all perform as expected. Also, edit the page counts for exempt and final page count and verify these display.
